apps/server/src/modules/lesson/service/lesson-copy.service.ts

```diff
@@ -224,18 +225,22 @@ export class LessonCopyService {
 				copiedContentStatus.push(embeddedTaskStatus);
 			}
 			if (element.component === ComponentType.NEXBOARD && nexboardEnabled) {
-				// eslint-disable-next-line no-await-in-loop
-				const nexboardContent = await this.copyNexboard(element, params);
 				const nexboardStatus = {
 					title: element.title,
 					type: CopyElementType.LESSON_CONTENT_NEXBOARD,
-					status: CopyStatusEnum.PARTIAL,
+					status: CopyStatusEnum.FAIL,
 				};
-				if (nexboardContent) {
-					copiedContent.push(nexboardContent);
-				} else {
-					nexboardStatus.status = CopyStatusEnum.FAIL;
+
+				if (nexboardCopyEnabled) {
+					// eslint-disable-next-line no-await-in-loop
+					const nexboardContent = await this.copyNexboard(element, params);
```

Comment:
This seems like a bad practice, please consider adjusting this eslint error
https://eslint.org/docs/latest/rules/no-await-in-loop

Reply:
Ah ok, I understand, thank you